product_docs/docs/pgd/5/routing/proxy.mdx

```diff
@@ -9,29 +9,36 @@ directoryDefaults:
   description: "The PGD Proxy is a service that acts as proxy layer between the client application and Postgres for your EDB Postgres Distributed cluster"
 ---
 
-EDB Postgres Distributed Proxy is a daemon that acts as an abstraction layer between the client application and Postgres. It interfaces with the BDR consensus mechanism to obtain the identity of the current write leader node and redirects traffic to that node.
+EDB Postgres Distributed Proxy is a daemon that acts as an abstraction layer between the client application and Postgres. It interfaces with the PGD consensus mechanism to get the identity of the current write leader node and redirects traffic to that node.
```

Comment:
Dee Dee didn't want me changing the fully spelled-out name to PGD. If you think it should be throughout, discuss that with Dee Dee. She wanted me to focus on not having BDR, since that is out-and-out wrong.